src/validations/valuesetValidation.ts

```diff
@@ -15,6 +15,20 @@ const GetValueSetErrors = async (
       valuesetsArray.map(async (valueSet) => {
         const oid = getOidFromCqlValueSet(valueSet, model);
         const locator = getLocatorFromCqlValueSet(valueSet);
+        if (model === "QICore") {
+          const valuesetUrlWithoutOid = valueSet?.url?.split("ValueSet/");
+          if (
+            valuesetUrlWithoutOid &&
+            valuesetUrlWithoutOid[0] !== "'http://cts.nlm.nih.gov/fhir/"
+          ) {
+            const invalidValuesetUrlError =
+              processValueSetErrorForElmTranslationError(
+                `"${valueSet.url} is not a valid URL"`,
```

Comment:
Do we want to enhance the error message suggesting what is the correct URL format? I think we need to ask POs or Michael. I don't see a down side by telling what is the accepted URL.

Reply:
thanks for the suggestion, I have updated the message based on the POs suggestion